lib/logstash/outputs/dynatrace.rb

```diff
@@ -199,11 +248,10 @@ def sleep_for_attempt(attempt)
       end
 
       def send_event(event, attempt)
```

Comment:
This is not actually one event but rather a serialized batch of events, right?

Reply:
Yes. It was before too actually. I didn't rename it because it came from the upstream http plugin and I wanted to keep them as similar as possible in case there were questions later and someone had to refer to the upstream code.

Reply:
Good point, then it's better to stay in sync with them, yeah. Thanks!